src/drt/src/io/io_parser_helper.cpp

```diff
@@ -592,138 +592,191 @@ inline void getTrackLocs(bool isHorzTracks,
   }
 }
 
+void io::Parser::checkFig(frPinFig* uFig,
+                          const frString& term_name,
+                          bool is_iTerm,
+                          const dbTransform& xform,
```

Comment:
is_iTerm is not needed. Just apply the transformation directly. if it's a bterm, just pass a default dbTransform which should not change the shape.